packages/account/src/Sections/Security/LoginHistory/login-history-table-row.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { Table, Text } from '@deriv/components';
+import { Localize } from '@deriv/translations';
+import { TLoginHistoryItems } from 'Types';
+
+const LoginHistoryTableRow = ({ id, date, action, browser, ip, status }: TLoginHistoryItems) => {
+    return (
+        <Table.Row className='login-history__table__row' key={id}>
+            <Table.Cell>
+                <Text line_height='xs' size='xs'>
+                    {date}
+                </Text>
+            </Table.Cell>
+            <Table.Cell className='login-history__table__row__cell login-history__table__row__cell--action'>
+                <Localize i18n_default_text='{{action}}' values={{ action }} />
+            </Table.Cell>
+            <Table.Cell>
+                {browser === 'Unknown' ? <Localize i18n_default_text='{{browser}}' values={{ browser }} /> : browser}
```

Comment:
Can we hardcode the browser value to to be `Unknown` here rather than passing it as a value?